README.md

```diff
@@ -112,10 +112,45 @@ Then, run the installation and remove the `.deb` file:
 ```bash
 dpkg -i /tmp/scw.deb && rm -f /tmp/scw.deb
 ```
+-->
 
 ## With a Docker Image
 
-For each release, we deliver a tagged image on the [Scaleway Docker Hub](https://hub.docker.com/r/scaleway/cli/tags) so can run `scw` in a sandboxed way: _Coming soon..._
+### Manual installation
```

Comment:
Is there a non manual way to install it with docker ? 

If so, maybe add the corresponding title before this section, with `Comming soo` as content.